components/style/properties/shorthand/font.mako.rs

```diff
@@ -73,6 +78,17 @@
             font_size: size,
             line_height: line_height,
             font_family: Some(font_family::SpecifiedValue(family))
+    % if product == "gecko":
+            ,
```

Comment:
nit: just keep this comma above. trailing commas are ok and often encouraged in rust